portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -96,6 +99,23 @@ private int getMaxCommentsToReturn(HttpServletRequest request) {
     return maxComments;
   }
 
+  /**
+   * Applies the sorting option specified by the user to the query.
+   */
+  private Query setSortOption(HttpServletRequest request, Query query) {
+    String sortOptionString = request.getParameter("sort-option");
```

Comment:
In terms of naming we try to omit words (e.g., "string") that are given based on the type.

(See https://drive.google.com/open?id=1-VB6otFQRR2SOESwAXRCHiHiZ4tx11iw hopefully you have access)